docs/integrating-with-relay.md

```diff
@@ -175,3 +175,104 @@ steps:
   - name: explicit-registry
     image: index.docker.io/relaysh/core
 ```
+
+## Designing a complete integration
+
+While Relay supports running any Linux container, it is most powerful when container images are accompanied by metadata that helps the service better interface the image requirements with users. We call the combination of metadata and image definitions a Relay integration.
+
+You may have already taken a look at the integrations we publish in [our GitHub organization](https://github.com/relay-integrations). Each of these is based on an initial [template](https://github.com/relay-integrations/template) that provides a starting point for the required parts of an integration:
+
+* A file called `integration.yaml` at the root of the integration that defines the integration name and version
+* For each step image, a file called `step.yaml` in the respective directory (for a step named `foo`, the path would be `steps/foo/step.yaml`)
+* For each trigger image, a file called `trigger.yaml` in the respective directory (for a trigger named `bar`, the path would be `triggers/bar/trigger.yaml`)
+
+For a deeper look at Relay integrations, take a look at the [integration layout RFC](https://github.com/puppetlabs/relay-rfcs/blob/master/content/0006-integration-layout/rfc.md).
+
+### Image metadata
+
+In addition to providing a name and version, the information in a `step.yaml` or `trigger.yaml` is consumed by Relay to give users more context about the image when they use it.
+
+Making sure to explicitly specify the following fields will help users get the best experience when they use your images:
+
+* `summary`: A human-readable title for the image, like "List EC2 instances"
+* `description`: A paragraph or two describing the purpose of the image and how to work with it in Markdown format
```

Comment:
Is this really a paragraph or two? I think the presentation of `description` in the app and website only exposes about 10 words/a sentence.